src/Elmish.WPF/ViewModelArgs.fs

```diff
@@ -27,13 +27,11 @@ module internal LoggingViewModelArgs =
       nameChain = "" }
 
 
-type internal ViewModelArgs<'model, 'msg> =
-  { initialModel: 'model
-    dispatch: 'msg -> unit
-    loggingArgs: LoggingViewModelArgs }
+type ViewModelArgs<'model, 'msg> =
```

Comment:
Keep this internal for now. Make in public in the PR that needs that.

Reply:
This essentially undoes this PR. I'll just close it until we implement static view models, which will need it.